mfs/dir.go

```diff
@@ -407,10 +415,14 @@ func (d *Directory) Path() string {
 }
 
 func (d *Directory) GetNode() (node.Node, error) {
+	return d.getNode(false)
+}
+
+func (d *Directory) getNode(clear bool) (node.Node, error) {
```

Comment:
There *has* to be a cleaner way to do this.

Reply:
yeah, i'm thinking about it. Its pretty obnoxious right now...

Reply:
That cross my mind also, but I didn't find it objectionable enough to comment. :)